src/include/miopen/conv/problem_description.hpp

```diff
@@ -367,6 +367,17 @@ struct ProblemDescription : ProblemDescriptionBase
     bool IsNCHWc_NCHWc() const;
     bool IsNCHWc_CHWNc() const;
 
+    bool HasNonPackedTensors() const
```

Comment:
[Recommendation] Formally speaking `HasNonPackedTensors` is incorrect when only one tensor is non-packed. I would say that `HasAtLeastOneNonPackedTensor` is fully correct, but I am not going to recommend this ;)

From the other hand, when we say "non-packed convolution" we imply that at least one tensor is non-packed, right? So maybe simply
```suggestion
    bool NonPacked() const
```
is better. Up to you; this is just a matter of taste.

Reply:
@atamazov : favoring the current name as it is more descriptive.

Reply:
[resolved]